src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -1039,7 +1039,7 @@ N2NTMToadlet.tooLarge=The file you have uploaded through your browser is ${attem
 N2NTMToadlet.sizeWarning=Please note that browser-based uploading has a limit of ${limit}.
 N2NTMToadlet.uploadFailed=Failed to upload file.
 N2NTMUserAlert.delete=Delete
-N2NTMUserAlert.header=From: ${from} (composed ${composed} | sent ${sent} | received ${received})
+N2NTMUserAlert.header=From: ${from} (sent ${sent})
```

Comment:
Composed might not be especially useful information, but I think both sent and received are. I know when I want to know about the timing of texts when I come out of a basement without cell coverage or something I'll want to know when it was sent, but if I come back to my phone to something like "ready to go in an hour" I'll want to know when it was received to know if I ever had a chance of responding in time. Thoughts?

I'm tired so this is too rambly but hopefully it makes sense anyway.